composer.json

```diff
@@ -65,7 +65,9 @@
         "symfony/maker-bundle": "^1.15",
         "rector/rector": "^0.8.15",
         "symfony/monolog-bundle": "^3.5",
-        "tracy/tracy": "^2.7"
+        "tracy/tracy": "^2.7",
+        "cweagans/composer-patches": "^1.7",
+        "migrify/vendor-patches": "^0.3.48"
```

Comment:
This is super cool trick for bugs in `/vendor`:

https://tomasvotruba.com/blog/2020/07/02/how-to-patch-package-in-vendor-yet-allow-its-updates/